[2024-06-24T11:40:39.744+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-24T11:40:39.771+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.processing_tasks.process_data_kyiv scheduled__2024-06-23T00:00:00+00:00 [queued]>
[2024-06-24T11:40:39.787+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.processing_tasks.process_data_kyiv scheduled__2024-06-23T00:00:00+00:00 [queued]>
[2024-06-24T11:40:39.788+0000] {taskinstance.py:2306} INFO - Starting attempt 4 of 4
[2024-06-24T11:40:39.822+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): processing_tasks.process_data_kyiv> on 2024-06-23 00:00:00+00:00
[2024-06-24T11:40:39.834+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1751) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-24T11:40:39.836+0000] {standard_task_runner.py:63} INFO - Started process 1778 to run task
[2024-06-24T11:40:39.836+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'processing_tasks.process_data_kyiv', 'scheduled__2024-06-23T00:00:00+00:00', '--job-id', '185', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpjw9sehvv']
[2024-06-24T11:40:39.840+0000] {standard_task_runner.py:91} INFO - Job 185: Subtask processing_tasks.process_data_kyiv
[2024-06-24T11:40:39.909+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_dag.processing_tasks.process_data_kyiv scheduled__2024-06-23T00:00:00+00:00 [running]> on host afc8096e8b86
[2024-06-24T11:40:40.055+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='processing_tasks.process_data_kyiv' AIRFLOW_CTX_EXECUTION_DATE='2024-06-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-23T00:00:00+00:00'
[2024-06-24T11:40:40.058+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-24T11:40:40.074+0000] {python.py:237} INFO - Done. Returned value was: (1719100800, 293.41, 79, 5, 2.39)
[2024-06-24T11:40:40.075+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-24T11:40:40.111+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=processing_tasks.process_data_kyiv, run_id=scheduled__2024-06-23T00:00:00+00:00, execution_date=20240623T000000, start_date=20240624T114039, end_date=20240624T114040
[2024-06-24T11:40:40.171+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-24T11:40:40.185+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
